src/test/resources/com/znsio/teswiz/features/vodqa.feature

```diff
@@ -6,4 +6,11 @@ Feature: Vodqa test
   Scenario: Validating scroll functionality using 2 points
     Given I login to vodqa application using valid credentials
     When I scroll from one to another element point on vertical swiping screen
-    Then Element text should be visible
\ No newline at end of file
+    Then Element text should be visible
+
+#  CONFIG=./configs/vodqa_local_config.properties PLATFORM=android TAG=vodqa ./gradlew run
```

Comment:
you can remove this as it is given at top already, so this is creating duplication